pkg/broker/services.go

```diff
@@ -68,11 +56,19 @@ func (b *Broker) buildServiceTemplate() (service domain.Service) {
 			LongDescription:     b.cfg.LongDescription,
 		},
 		PlanUpdatable: true,
-		Plans:         b.buildPlansForProviderDynamic(),
+		Plans:         b.buildPlans(),
+	}
+
+	for _, p := range svc.Plans {
```

Comment:
[q] are these changes relevant for fixing binding?

Reply:
No, just some housekeeping